prow/plugins/label/label_test.go

```diff
@@ -177,7 +176,6 @@ func TestLabel(t *testing.T) {
 			expectedNewLabels:     formatLabels(),
 			expectedRemovedLabels: []string{},
 			commenter:             orgMember,
-			expectedBotComment:    true,
```

Comment:
Do we care to comment in the cases we've removed here? Technically, we could give feedback in these cases, it would just be tailored  to the error. For instance, it might be nice to tell someone they can't use `/label` unless they're in the org, or that they're asking for an area that does not exist (and show which ones do).

Reply:
Let's follow this PR up with another to add those in, or make an issue if you don't think you want to handle those cases yourself!